src/js/components/modals/AppModalComponent.jsx

```diff
@@ -115,6 +141,10 @@ var AppModalComponent = React.createClass({
     }
   },
 
+  hasInfoObject: function () {
```

Comment:
No "store" in sight.
Just a convenient function, checking against an object in the state.
I don't want to pass the state into a private function.